src/buildfromsource/FSharp.Compiler.Private/FSComp.resx

```diff
@@ -4231,7 +4231,7 @@
     <value>Used instead of let for values that require Dispose to be called to free resources.</value>
   </data>
   <data name="keywordDescriptionUseBang" xml:space="preserve">
-    <value>Used instead of let! in asynchronous workflows and other computation expressions for values that require Dispose to be called to free resources.</value>
+    <value>Used instead of let! in computation expressions for values that require Dispose to be called to free resources.</value>
```

Comment:
@jwosty, perhaps this should be:

"Used instead of let! in computation expressions for values that implement IDisposable"
It's technically mor accurate, and IDisposable is sometimes used merely to defer execution until after a block of code has executed.

Reply:
I like that. I should do the same for the `use` keyword too, then

Reply:
Or perhaps even more accurate would be to say "Used instead of let! in computation expressions for computation expression results that implement IDisposable."?

Reply:
@jwosty I was wondering if you are going to commit the discussed textual change, or should I take care of it?

Thanks

Kevin

Reply:
I intended on it but I've been pretty busy. If you want to do it that's fine, since I can't guarantee when I'll be able to get around to it. :)